terraform/main.tf

```diff
@@ -220,6 +220,19 @@ resource "aws_ses_receipt_rule" "ffis_ingest" {
     position          = 1
     object_key_prefix = "ses/ffis_ingest/new"
   }
+
+  count = var.version_identifier == "dev" ? 0 : 1
```

Comment:
I added this line to prevent it from failing to plan, locally at least. Let me know if/when I should remove @TylerHendrickson @slapula.

Reply:
Should be okay to remove this now!